src/main/java/com/tcs/test/FlipkartAddToCartFlow.java

```diff
@@ -0,0 +1,69 @@
+package com.tcs.test;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+
+import com.tcs.driver.Driver;
+
+public class FlipkartAddToCartFlow {
+	public static WebDriver driver=null;
+	
+	public static void main(String[] args) throws InterruptedException {
+		 
+		/**
+		 * Get the driver and navigate to Flipkart
+		 * this approach will help us to use multiple drivers
+		 */
+		driver =Driver.getChromeDriver();	
+		driver.manage().window().maximize();
+     	driver.navigate().to("https://www.flipkart.com");
+     	
+     	/**
+     	 *This method is used to login on the website
+     	 */
+		login("9015677021","Flipkart@123");
+		
+		/**
+		 * This method is for loging out form the site
+		 */
+		logout();
+		
+	}
+
+	private static void logout() throws InterruptedException {
+		
+		Thread.sleep(1000);
```

Comment:
Search on google how you can wait for a particular web element (e.g `div [class=exehdJ])`to be present/visible before finding it using css or xpath.
We should always try to avoid using `Thread.sleep()` as it will always wait for the specified time , what if the element appeared before that, its extra time that we will be waiting.
Search on google about `implicit` and `explicit waits` in Selenium and try to implement them here. We will be using explicit wait here, just an hint.